util/unifyfs/src/unifyfs-rm.c

```diff
@@ -271,6 +368,27 @@ static int remove_server_pid_file(unifyfs_args_t* args)
     return ret;
 }
 
+static int remove_stage_status_file(unifyfs_args_t* args)
+{
+    int ret = 0;
+    char filename[PATH_MAX] = { 0, };
```

Comment:
No need to zero this array

Reply:
I've removed the zero initalizations for all the buffers in unifyfs-rm.c that defined buffers that were written directly to in that function by string (and thus NULL terminating) functions.